ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -118,9 +118,9 @@
   "Get the concept-ids from either the concept-ids passed in or 
    from the provider. If both are empty, throws exception." 
   [context concept-ids provider-id]
-  (if-let [concept-ids (if (seq concept-ids)
-                         concept-ids
-                         (get-provider-concept-ids context provider-id))]
+  (if-let [concept-ids (if (= "[\"ALL\"]" (string/trim (string/upper-case concept-ids)))
```

Comment:
This check looks off. It should just check `(map string/upper-case concept-ids)` equals `["ALL"]`. Also what if concept-ids is `["C1111-PROV1" "ALL"]`? I don't see this edge case being addressed.

Reply:
We should add a test for this case.

Reply:
I don't need to.  Honestly I thought about the case and when the user is mixing "ALL" with other concept-ids, it will be treated as the regular concept-ids. Only when the concept-ids = ["ALL"], it's treated as all collections.

Reply:
In which case, it will complain "ALL" concept-id doesn't exist.

Reply:
I think we should add validation of the concept-ids, any invalid concept-id will fail the request (including the mixed case). Only `["ALL"]` will be accepted as valid option for all collections.

Reply:
Done.